example/Python/src/DigitalTwin/KeyFob/KeyFobDemo.lf

```diff
@@ -0,0 +1,113 @@
+target Python {
```

Comment:
A short description here at the top would be helpful I think with a pointer to the README file.